src/pystack/_pystack/pyframe.cpp

```diff
@@ -62,7 +62,12 @@ FrameObject::getCode(
                << py_code_addr;
 
     uintptr_t last_instruction;
-    if (manager->versionIsAtLeast(3, 11)) {
+    if (manager->versionIsAtLeast(3, 13)) {
+        // In 3.13, the field is really called *instr_ptr*, but we are reusing the same
+        // field name as in 3.12 to avoid having to change the code in multiple places.
```

Comment:
This doesn't explain the most surprising bit:
```suggestion
        // In 3.13 the field is called *instr_ptr* and points to the current instruction
        // rather than the previous one. We're reusing the same name and adjusting the
        // value to allow us to use the same decoding code across versions.
```